api/v3/Activity.php

```diff
@@ -372,6 +407,13 @@ function civicrm_api3_activity_get($params) {
       }
     }
   }
+
+  // Ensure there's enough data for calculating is_overdue
+  if (!empty($options['return']['is_overdue']) && (empty($options['return']['status_id']) || empty($options['return']['activity_date_time']))) {
+    $options['return']['status_id'] = $options['return']['activity_date_time'] = 1;
```

Comment:
So, does this mean that if I call the API like this I will still get the `activity_date_time` and `status_id` fields, even though I'm explicitly saying I only want the is_overdue value?:

```php
civicrm_api3('Activity', 'get', [
  'return' => ['is_overdue']
])
```